pkgs/by-name/mg/mgitstatus/package.nix

```
@@ -0,0 +1,37 @@
+{ fetchFromGitHub, lib, mgitstatus, stdenv, testers }:
```

```suggestion
{ fetchFromGitHub
, lib
, mgitstatus
, stdenv
, testers
}:
```

Small nit to make diffing easier in the future

---

Still feels weird to do this for line lengths <80 haha.